guides/hack/21-XHP/07-basic-usage.md

```diff
@@ -63,12 +63,12 @@ $xhp = <h:p>Hello, world</h:p>;
 ```
 
 ```
-// from global namespace:
+// exists in the root namespace:
 $xhp = <Facebook:XHP:HTML:p>Hello, world</Facebook:XHP:HTML:p>;
 ```
 
 ```
-namespace CustomNamespace;  // from any namespace:
+namespace CustomNamespace;  // from a defined namespace:
```

Comment:
The original `any` had meaning here. A name the begins with a namespace separator is fully qualified. These names unambiguously refer to the same thing, whether in the root namespace or CustomNamespace.

Reply:
This is one of those changes I meant to reverse but accidentally left in the commit. Agreed.